src-docs/src/components/guide_section/_utils.js

```diff
@@ -41,8 +40,8 @@ export const renderJsSourceCode = (code) => {
     // [\r\n]                  - match end of line, so the extra new line is removed via the replace operation
     /import {([^}]+)} from '@elastic\/eui';[\r\n]/g,
     (match, imports) => {
-      // remove any additional characters from imports
-      const namedImports = imports.match(/[a-zA-Z0-9]+/g);
+      // remove all characters except (_) from the imports
```

Comment:
optional comment clarification
```suggestion
      // remove all characters that aren't letters, numbers, or underscores from the imports
```